pipettes/core/CMakeLists.txt

```diff
@@ -1,41 +1,47 @@
-function(target_pipettes_core_common TARGET)
+function(target_pipettes_core_common TARGET REVISION)
     target_sources(${TARGET} PUBLIC
             ${CMAKE_CURRENT_FUNCTION_LIST_DIR}/configs.cpp
             ${CMAKE_CURRENT_FUNCTION_LIST_DIR}/peripheral_tasks.cpp
             ${CMAKE_CURRENT_FUNCTION_LIST_DIR}/central_tasks.cpp
             ${CMAKE_CURRENT_FUNCTION_LIST_DIR}/linear_motor_tasks.cpp
-            ${CMAKE_CURRENT_FUNCTION_LIST_DIR}/gear_motor_tasks.cpp
-            ${CMAKE_CURRENT_FUNCTION_LIST_DIR}/sensor_tasks.cpp)
+            ${CMAKE_CURRENT_FUNCTION_LIST_DIR}/gear_motor_tasks.cpp)
+    if (${REVISION} STREQUAL "rev1")
```

Comment:
we might make further revisions; proto would be the odd one out. maybe invert this so it's `if (${REVISION} STREQUAL "proto") ... else` and that way if we make a `rev2` we won't have to change it